mne/source_estimate.py

```diff
@@ -2047,8 +2061,10 @@ class VectorSourceEstimate(_BaseVectorSourceEstimate,
     data : array of shape (n_dipoles, 3, n_times)
         The data in source space. Each dipole contains three vectors that
         denote the dipole strength in X, Y and Z directions over time.
-    vertices : array | list of shape (2,)
-        Vertex numbers corresponding to the data.
+    vertices : list of array, shape (2,)
+        Vertex numbers corresponding to the data. The first element of the list
+        contains vertices of left hemisphere and the second element contains
+        vertices of right hemisphere.
```

Comment:
I am a bit confused here @agramfort . When is it array and when is it list of array? Or is always array?

Reply:
vertices is always a list of arrays and its length in the number of pieces in the source space (eg 2 for surface, 1 for volume grid etc.)